unit_tests/test_cli.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import webbrowser
+import os
+import shutil
+from robyn.__main__ import create_robyn_app, docs
+
+
+class TestCreateRobynApp(unittest.TestCase):
```

Comment:
This is more of a system test than a unit test. And creating a file is a bit flakey if run multiple times.

We can either use `mocking`(https://docs.python.org/3/library/unittest.mock.html) or move this to integration_tests and use `fixtures` for setup and teardown.

What would you prefer @Mr-Sunglasses ? I would suggest a mocking-based approach.

Reply:
@sansyrox I would also prefer the mocking-based approach 😄